discoveryengine/standalone_apis_sample.py

```diff
@@ -42,17 +42,24 @@ def check_grounding_sample(
         answer_candidate="Titanic was directed by James Cameron. It was released in 1997.",
         facts=[
             discoveryengine.GroundingFact(
-                fact_text=(
-                    "Titanic is a 1997 American epic romantic disaster movie. It was directed, written,"
-                    " and co-produced by James Cameron. The movie is about the 1912 sinking of the"
-                    " RMS Titanic. It stars Kate Winslet and Leonardo DiCaprio. The movie was released"
-                    " on December 19, 1997. It received positive critical reviews. The movie won 11 Academy"
-                    " Awards, and was nominated for fourteen total Academy Awards."
-                ),
+                fact_text="""Titanic is a 1997 American epic romantic disaster movie.
```

Comment:
See https://google.github.io/styleguide/pyguide.html#310-strings. This style with triple quotes would add the leading whitespace.

Reply:
Thanks for the feedback.